spec/components/breadcrumbs_spec.rb

```diff
@@ -128,14 +128,16 @@ def assert_link_with_text_in(selector, link, text)
     assert_select ".gem-c-breadcrumbs--inverse"
   end
 
-  it "ignore breadcrumb items without url" do
+  it "renders breadcrumb items without link as  text" do
     render_component(
       breadcrumbs: [
         { title: 'Topic', url: '/topic' },
         { title: 'Current Page' },
       ]
     )
-    assert_select('.govuk-breadcrumbs__list-item:last-child', 'Topic')
+
+    assert_link_with_text_in('.govuk-breadcrumbs__list-item:first-child', '/topic', 'Topic')
+    assert_select('.govuk-breadcrumbs__list-item:last-child', 'Current Page')
```

Comment:
Does this test actually prove there's no link in the final list item? Or do we need to be more explicitly checking there's no link element?